examples/vite/src/index.scss

```
@@ -7,12 +7,14 @@
 @import url('modern-normalize') layer(modern-normalize);
 @import url('stream-chat-react/dist/css/index.css') layer(stream-new);
 @import url('./AppSettings/AppSettings.scss') layer(stream-app-overrides);
+@import url('./AppSettings/ActionsMenu/DraggableDialog.scss') layer(stream-app-overrides);
```

Stylelint (17.14.0)

[error] Expected "url('./AppSettings/ActionsMenu/DraggableDialog.scss')" to be "'./AppSettings/ActionsMenu/DraggableDialog.scss'" (import-notation)

(import-notation)